colossus-examples/src/main/scala/colossus.examples/BenchmarkExample.scala

```diff
@@ -3,30 +3,24 @@ package colossus.examples
 import colossus.IOSystem
 import colossus.service.Callback.Implicits._
 import colossus.protocols.http._
-
-import org.json4s._
-import org.json4s.jackson.JsonMethods._
-import org.json4s.JsonDSL._
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
 
 object BenchmarkService {
 
-  implicit object JsonBody extends HttpBodyEncoder[JValue] {
-    val contentType = ContentType.ApplicationJson
+  private val mapper: ObjectMapper = new ObjectMapper().registerModule(DefaultScalaModule)
 
-    def encode(json: JValue) = {
-      HttpBody(compact(render(json)))
-    }
+  implicit object JsonBody extends HttpBodyEncoder[Array[Byte]] {
+    val contentType: String = ContentType.ApplicationJson
+    def encode(json: Array[Byte]): HttpBody = new HttpBody(json)
   }
 
-  val json: JValue = ("message" -> "Hello, World!")
+  def json: Array[Byte] = mapper.writeValueAsBytes(Map("message" -> "Hello, World!"))
```

Comment:
Is there no JObject we can create first? The `writeValueAsBytes` logic would make more sense in the encode method.